newIDE/app/src/UI/Toolbar.js

```diff
@@ -48,6 +48,7 @@ export const Toolbar = React.memo<ToolbarProps>(
           // Use content-visibility as we know the exact height of the toolbar, so the
           // content can be entirely skipped when hidden:
           contentVisibility: hidden ? 'hidden' : 'visible',
+          pointerEvents: hidden ? undefined : 'all',
```

Comment:
should it be "none" instead of undefined?

Reply:
I'll keep undefined because I prefer to avoid adding a styling prop if we can, especially these ones about events which can be tricky.